tests/databrowser.js

```diff
@@ -157,40 +156,13 @@ define([
                 });
             });
 
-            it("should add recursive ref on drag/drop", function(done) {
```

Comment:
What happens if vellum gets a schema with recursive relationships now? Will it go into an infinite loop trying to unwind them?